examples/with-tailwindcss/package.json

```
@@ -6,13 +6,18 @@
     "start": "next start"
   },
   "dependencies": {
-    "next": "latest",
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2"
+    "next": "12.0.7",
+    "react": "17.0.2",
+    "react-dom": "17.0.2"
```

```suggestion
    "next": "latest",
    "react": "^17.0.2",
    "react-dom": "^17.0.2"
```

I don't see a reason why this should change.